src/app/modules/main/chat_section/module.nim

```diff
@@ -582,12 +591,12 @@ method addNewChat*(
   elif chatDto.chatType == ChatType.PrivateGroupChat:
     chatImage = chatDto.icon
 
-  var amIChatAdmin = self.amIMarkedAsAdminUser(chatDto.members)
-  if not amIChatAdmin and len(chatDto.communityId) != 0:
-    let community = communityService.getCommunityById(chatDto.communityId)
-    amIChatAdmin = amIChatAdmin or community.admin
+  var memberRole = self.getUserMemberRole(chatDto.members)
+  
+  if memberRole != MemberRole.None and len(chatDto.communityId) != 0:
```

Comment:
`not amIChatAdmin` changed to `memberRole != MemberRole.None`
Is this correct? 🤔

Reply:
nice catch, fixed